code/modules/virtual_reality/remote_mech.dm

```diff
@@ -0,0 +1,31 @@
+/mob/living/carbon/human/proc/mech_selection(var/list/network)
```

Comment:
See if you can move that proc to the chair (instead of having it at carbon/human)